utilities/OBAAnalytics.m

```diff
@@ -32,6 +32,10 @@ + (void)reportScreenView:(NSString *)label {
 
 + (void)reportViewController:(UIViewController*)viewController {
     [self reportScreenView:[NSString stringWithFormat:@"View: %@", viewController.class]];
+    
+    if (UIAccessibilityIsVoiceOverRunning()) {
+        [OBAAnalytics reportEventWithCategory:OBAAnalyticsCategoryAccessibility action:@"voiceover_on" label:[NSString stringWithFormat:@"Loaded view: %@ using VoiceOver", viewController.class] value:nil];
```

Comment:
Another option may be to just use a custom dimension that keeps track of if voice over is running or not. This would also allow one to see which features are used with voice over vs. which are not.